### DISCUSSION TITLE
Program crashing upon launch with Windows XP Service Pack 3 in VirtualBox

### DISCUSSION THREAD

**Discussion by seedy60** [LegacyUpdate/LegacyUpdate#61] [Questions & Help]
Hi,
I just tried using Legacy Update on a fresh install of Windows XP SP3 in VirtualBox. When I launch the Legacy Update executable, however, it simply crashes and refuses to run at all.
Legacy Update has encountered a problem and needs to close. We are sorry for the inconvenience.
In Windows 2000 SP4, which I also have running in VirtualBox, the tool works completely fine. What could be going on here?
Note: this is a completely fresh install of Windows XP, with nothing installed besides my screen reader.

**Comment by kirb** [comment]
This is currently known but I haven’t had a chance to look into it yet. See #37 and #45 for the discussions on it at the moment. Workaround is to use version 1.4, though that will have error 82 when installing on XP Home, which is what I was trying to fix with 1.4.1.

**Reply by seedy60** [reply]
I'm using Professional, so no error 82 for me! :)
